platforms/unix/vm/sqUnixMain.c

```diff
@@ -1418,13 +1420,8 @@ static int vm_parseArgument(int argc, char **argv)
       return 1;
     }
 
-  /* legacy compatibility */		/*** XXX to be removed at some time ***/
 
-#ifdef PharoVM
-# define VMOPTION(arg) "--"arg
-#else
 # define VMOPTION(arg) "-"arg
-#endif
 
```

Comment:
I removed the #ifdef because dash prefix is not a compile time option. Modules have their own parser for arguments :-(. While #ifdef seems like a simple fix for the core vm, it complicates argument parsing in modules, particularly those which are shared between Squeak and Pharo :-(. It also affects hard-coded messages like the one in imageNotFound().